arkouda/message.py

```diff
@@ -123,10 +123,16 @@ def _build_list_param(key: str, val: list) -> ParameterObject:
 
         # want the object type. If pdarray the content dtypes can vary
         dtypes = {type(p).__name__ for p in val}
-        if len(dtypes) > 1:
-            t_str = ", ".join(dtypes)
-            raise TypeError(f"List values must be of the same type. Found {t_str}")
-        t = dtypes.pop()
+        if len(dtypes) == 1:
+            t = dtypes.pop()
+        else:
+            for t in dtypes:
+                if t not in [pdarray.__name__, Strings.__name__]:
+                    t_str = ", ".join(dtypes)
+                    raise TypeError(f"Lists of multiple types can only "
+                                    f"conatin strings and pdarray. Found {t_str}")
```

Comment:
The logic looks right to me, just a small spelling mistake with "contain".

Reply:
fixed